README.md

```diff
@@ -73,6 +73,19 @@ Or pass a locator to test part of the page:
 await expect(page.locator('#my-element')).toBeAccessible()
 ```
 
+You can also pass an [Axe results
+object](https://www.deque.com/axe/core-documentation/api-documentation/#results-object)
+to the matcher:
+
+```js
+import { runAxe } from 'expect-axe-playwright'
+
+test('should be accessible', async ({ page }) => {
+  const { results } = runAxe(page)
+  await expect(results).toBeAccessible()
```

Comment:
If we want the matcher to support an Axe results object and for the resulting line of code to read in a semantically coherent and accurate way, it may make sense to [change the matcher name](https://github.com/Widen/expect-axe-playwright/pull/24). That way this would change to the more sensical:

```js
expect(results).toHaveNoAxeViolations()
```